bot/bot.js

```diff
@@ -44,6 +47,9 @@ bot.on("ready", function() {
   require("./plugins.js").init();
   console.log("type " + config.prefix + "help in Discord for a commands list.");
   bot.user.setGame(config.prefix + "help");
+
+  //initialize the claimbot (content bot)
+  claimbot.init(bot);
```

Comment:
Init will be called automatically when initziating the bot from the plugin loader as long as init is in a customFunctions export, refer to here for more information: https://github.com/lbryio/lbry-wunderbot/blob/master/bot/modules/example.js :)